gateways/mercurius/install.sh

```diff
@@ -0,0 +1 @@
+yarn install
```

Comment:
why yarn? `npm` is available by default if somebody installs `node`, `yarn` is not

Reply:
replaced with `npm`.